layer/logged_data/log_data_runner.py

```diff
@@ -73,7 +73,7 @@ def log(
             elif isinstance(value, bool):
                 self._log_boolean(tag=tag, bool_val=value)
             elif isinstance(value, (int, float)):
-                if self._train_id and epoch is not None:
```

Comment:
Metrics are only meant for models, so this was correct as is. Please revert.

Reply:
Done